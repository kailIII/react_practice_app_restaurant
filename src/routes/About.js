import React, { Component } from 'react';
import {Link} from 'react-router-dom';

class Home extends Component {
  render() {
    return (
        <div className="Appstuff">
          <h1>A bit about us: </h1>
          <h2>Staff</h2>

          <h4>Brad Thomas</h4>
          <ul>
            <li>Owner/CEO</li>
            <li>MBA Harvard University </li>
          </ul>

          <h4>Chef Tony</h4>
          <ul>
            <li>Lead chef</li>
            <li>Graduate of New York Culinary Institute</li>
          </ul>

          <h4>Chef Victor</h4>
          <ul>
            <li>Sioux Chef</li>
            <li>Graduate of New York Culinary Institute</li>
          </ul>

          <h4>Ana Rodriguez</h4>
          <ul>
            <li>Head of Marketing and Operations</li>
            <li>MBA University of New York</li>
          </ul>

          <Link to="/">Home</Link>
        </div>
    );
  }
}

export default Home;
