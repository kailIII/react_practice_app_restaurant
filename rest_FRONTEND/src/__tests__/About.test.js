import React from 'react';
import ReactDOM from 'react-dom';
import { MemoryRouter } from 'react-router-dom';
import About from '../routes/About';
import renderer from 'react-test-renderer';
import { mount, shallow } from 'enzyme';
import sinon from 'sinon';


it('should render', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <About/>
    </MemoryRouter>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
