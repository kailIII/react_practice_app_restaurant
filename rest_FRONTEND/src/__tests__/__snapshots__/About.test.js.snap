// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="Appstuff"
>
  <h1>
    A bit about us: 
  </h1>
  <h2>
    Staff
  </h2>
  <h4>
    Brad Thomas
  </h4>
  <ul>
    <li>
      Owner/CEO
    </li>
    <li>
      MBA Harvard University 
    </li>
  </ul>
  <h4>
    Chef Tony
  </h4>
  <ul>
    <li>
      Lead chef
    </li>
    <li>
      Graduate of New York Culinary Institute
    </li>
  </ul>
  <h4>
    Chef Victor
  </h4>
  <ul>
    <li>
      Sioux Chef
    </li>
    <li>
      Graduate of New York Culinary Institute
    </li>
  </ul>
  <h4>
    Ana Rodriguez
  </h4>
  <ul>
    <li>
      Head of Marketing and Operations
    </li>
    <li>
      MBA University of New York
    </li>
  </ul>
  <a
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
</div>
`;
